src/types.ts

```diff
@@ -240,6 +240,12 @@ export interface Instance {
      */
     uploadMedia(media: Buffer): Promise<string>;
 
+    startUrlUploadJob(urls: string[]): Promise<UrlToJobId[]>; // returns job ids
```

Comment:
These return values are wrong, this is saying that the return type will be an array, but it's not.